### DISCUSSION TITLE
Unused GPT 4 Account?

### DISCUSSION THREAD

**Discussion by outrcore** [community/community#53335] [Apps, API and Webhooks]
If you have an OpenAI account with GPT4 API (API Key) access, I'd be interested in putting that account to good use. Shoot a reply here or DM me on Twitter @Mythical_Soup if you're willing to part with the account, don't have much but would be willing to negotiate if need. Thank you!

**Comment by queenofcorgis** [comment]
Hi @outrcore , we’re glad you’re here 😀

Thanks for bringing this discussion to the community. Unfortunately, we currently do not allow bartering on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members. I'm going to close and lock this post.
